docs/contributing.md

```diff
@@ -1,15 +0,0 @@
-# 👭 Contributing
```

Comment:
Do we really want remove this page?

Reply:
I included it in the README again in a shortened format. That will be visible on the docs website as well.